larky/src/test/resources/vendor_tests/pycryptodome/Crypto/Cipher/test_CFB.star

```diff
@@ -0,0 +1,31 @@
+load("@stdlib//builtins","builtins")
+load("@stdlib//base64", "base64")
+load("@stdlib//unittest","unittest")
+load("@vendor//Crypto/Cipher/AES", AES="AES")
+load("@vendor//asserts","asserts")
+
+def b(s):
+    return builtins.bytes(s, encoding="utf-8")
+
+eq = asserts.eq
+
+def CfbTests_test():
```

Comment:
You will need a lot more comprehensive test suite than this. There are usually many edge cases that we have to run through but I think this is a good first pass.

Reply:
added more tests